src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -384,9 +381,9 @@ function InitializeFirebaseFunctionsStubs(functionsDir: string): void {
     failing in some way and admin is attempting to access prod resources. This error isn't pretty,
     but it's hard to catch and better than accidentally talking to prod.
    */
-function InitializeFirebaseAdminStubs(frb: FunctionsRuntimeBundle): typeof admin {
-  const adminResolution = slowRequireResolve("firebase-admin", frb.cwd);
-  const grpc = require(slowRequireResolve("grpc", frb.cwd));
+async function InitializeFirebaseAdminStubs(frb: FunctionsRuntimeBundle): Promise<typeof admin> {
```

Comment:
One day I will be a big boi who knows JavaScript and will understand this `typeof` thing.

Reply:
False.